deploy/sre-build-test/50-build-test.CronJob.yaml

```diff
@@ -55,15 +59,17 @@ spec:
               trap "cleanup" EXIT SIGINT
 
               # set NS to include job name, to ease linking namespace to a specific job
-              export NS="openshift-build-test-${JOB_NAME##sre-build-test-}"
+              export CRONJOB_NAME=sre-build-test
+              export NS="${JOB_NS}-${JOB_NAME##${CRONJOB_NAME}-}"
               oc create ns "${NS}"
 
               # run build
-              oc -n "${NS}" new-build nodejs~https://github.com/openshift/nodejs-ex --name=sre-build-test
+              oc -n "${NS}" new-build nodejs~https://github.com/openshift/nodejs-ex --name="${CRONJOB_NAME}"
               echo "$(date): Waiting for build to complete."
               while :
               do
-                export ST=$(oc -n "${NS}" get build -o custom-columns=STATUS:.status.phase --no-headers)
+                # fixes shellcheck error SC2155
```

Comment:
```suggestion
```

Reply:
do we actually need to export ST?

Reply:
I don't think any of them need to be exported